citation/caching.py

```diff
@@ -31,7 +37,7 @@ def initialize_contributor_cache():
             temp = {}
             for count in contributor_count:
                 if count['id'] == log['id']:
-                    temp.update({'id': log['id'], 'contribution': log['contribution'] * 100 / count['count'],
+                    temp.update({'id': log['id'], 'contribution': "{0:.2f}".format(log['contribution'] * 100 / count['count']),
```

Comment:
if you're using update with keys that are also valid Python identifiers I would recommend using this form instead:
`temp.update(id=log['id'], contribution='...')`